Guide.md

```diff
@@ -10,10 +10,18 @@ The plugin allows the user to easily import data from DMIs Open Data from Meteor
 
 ## Installation
 
-Open QGIS and go to Plugins and then Manage and Install Plugins. Search for DMI Open Data and install the plugin. The plugin will be located in the plugin toolbar.
+Open QGIS and go to Plugins and then Manage and Install Plugins. ![Manage Plugins](guide%20image/plugin_installer.png)  
+Search for DMI Open Data and install the plugin. The plugin will be located in the plugin toolbar.
 
 ![Installation](guide%20image/installl.png)
 
+## Activate your API keys
```

Comment:
Suggest Enter/Insert instead of Activate.

Activate makes me think of the signup flow and not the process of typing (or copy/pasting) them into the plugin.